src/transformers/tokenization_utils_base.py

```diff
@@ -2890,6 +2890,13 @@ def _call_one(
         split_special_tokens: bool = False,
         **kwargs,
     ) -> BatchEncoding:
+        # Check if is_split_into_words is used with add_prefix_space
+        if is_split_into_words and hasattr(self, "add_prefix_space") and not getattr(self, "add_prefix_space", False):
+            raise ValueError(
+                f"You need to instantiate {self.__class__.__name__} with add_prefix_space=True "
+                "to use it with pretokenized inputs."
+            )
```

Comment:
Is this specific to ModernBert or does that fix all tokenizers? because add_prefix_space is not something all tokenizer suppport!